tar/bsdtar.c

```diff
@@ -1008,11 +1017,6 @@ main(int argc, char **argv)
 		break;
 	}
 
-	cleanup_exclusions(bsdtar);
```

Comment:
oh, in case there's any doubt: yes, moving that `cleanup_exclusions(bsdtar);` would be a change from libarchive code.  (see my earlier comment for why I still think it's appropriate in this case)